Gopkg.toml

```diff
@@ -35,3 +35,7 @@
 [[constraint]]
   name = "github.com/etcd-io/gofail"
   branch = "master"
+
+[[constraint]]
+  name = "github.com/pkg/errors"
```

Comment:
seem we have already used juju errors, conflicted?

Reply:
What is the conflict? Both packages work together fine by using the same `Causer` interface. The rest of the API is the same, but with slightly different names (but pkg/errors can record a stack trace). See: 
 
 * your issue for switching https://github.com/pingcap/tidb/issues/2622
 * a more detailed issue https://github.com/pingcap/tidb/issues/7125
 * the PR to make the switch https://github.com/pingcap/tidb/pull/7151

I am happy to send a PR to make the switch in PD when we are ready.

Reply:
@gregwebs 

I suggest only using one errors, so it is better to remove juju at first.

Reply:
Okay, I can send a PR for that. Would you like to continue to review this PR in the mean time since that PR won't affect this one (other than both adding pkg/errors in Gopkg)?

Reply:
of course, we can go on

Reply:
#1210 is merged: we are now using pkg/errors